.github/workflows/tests.yml

```diff
@@ -240,6 +240,7 @@ jobs:
 
           composer update
           composer exec -- drush updb -y
+          composer exec -- drush pm:enable thunder_news_article --yes
```

Comment:
Why do we need this? Shouldn't it be enough to enable the module in the test base classes?